aea/cli/add.py

```diff
@@ -41,6 +41,20 @@ def add(ctx: Context):
     _try_to_load_agent_config(ctx)
 
 
+@add.command()
+@click.argument('connection_name', type=str, required=True)
+@click.argument('connection_type', type=click.Choice(['oef', 'local', 'gym']), required=True)
+@click.option('--config', nargs=2, multiple=True, metavar='KEY VALUE', help='Overrides a config key/value pair.')
```

Comment:
I think I said before. Not a fan of this at all. We should have it all via YAML configs in my opinion to keep the CLI neat and clean. I don't see the use case for this once you have the yaml config.